arangod/Replication2/ReplicatedLog/LogStatus.h

```diff
@@ -134,4 +135,16 @@ struct LogStatus {
   VariantType _variant;
 };
 
+/**
+* @brief Provides a more general view of what's currently going on, without
+* completely relying on the leader.
+ */
+struct GlobalStatus {
+  agency::LogCurrentSupervision supervision;
+  std::optional<LogStatus> logStatus;
```

Comment:
I would have expected this to be a `unordered_map<ParticipantId, LogStatus>`. Hence you would get something like this:
```{
   "supervision": {...},
   "participants": { // only leader
     "PRMR-xxxx": { /* local status */ },
     // in future on could add the other participants here as well
   }
   "leaderId": "PRMR-xxxx",
}
```